open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/open-lineage-connectors/open-lineage-janus-connector/src/main/java/org/odpi/openmetadata/openconnectors/governancedaemonconnectors/openlineageconnectors/janusconnector/graph/LineageGraphConnectorHelper.java

```diff
@@ -728,7 +735,7 @@ private void addColumnProperties(Set<LineageVertex> lineageVertices) {
             return;
         }
         GraphTraversalSource g = graphFactory.getGraphTraversalSource();
-        lineageVertices.stream().filter(this::needsAdditionalNodeContext).forEach(lineageVertex -> {
+        lineageVertices.forEach(lineageVertex -> {
```

Comment:
this check should be kept for ultimate source/ ultimate destination nodes or a special check for those types of nodes.